src/dynamo0p3.py

```diff
@@ -2949,7 +2960,18 @@ def generate(self):
         # Add boundary dofs array to the operator boundary condition
         # kernel (enforce_operator_bc_kernel) arguments
         if self._kern.name.lower() == "enforce_operator_bc_code":
-            self.operator_bcs_kernel(fs_to)
+            # Minimal sanity check - this kernel should only have a single LMA
+            # operator as argument so the first one must be a gh_operator.
+            # Full sanity checking is done in the operator_bcs_kernel()
```

Comment:
Having talked this through with Rupert we have agreed that methods such as `operator_bcs_kernel()` should be private. I've created ticket #36 for this.
I'm now happy with putting the checks into `generate()` so will do that.